samples/basic/minimal/common.conf

```diff
@@ -0,0 +1,35 @@
+# Drivers and peripherals
+CONFIG_I2C=n
+CONFIG_WATCHDOG=n
+CONFIG_GPIO=n
+CONFIG_SPI=n
+CONFIG_SERIAL=n
+CONFIG_FLASH=n
+
+# Power management
+CONFIG_SYS_POWER_MANAGEMENT=n
+
+# Interrupts
+CONFIG_DYNAMIC_INTERRUPTS=n
+CONFIG_IRQ_OFFLOAD=n
+
+# Memory protection
+#CONFIG_ARM_MPU=n
```

Comment:
This is, actually, ARM-specific - I wonder if we need to be selecting (the common) MEMORY_PROTECTION directly, and depreciate entirely the "ARCH"_MPU symbols.

Reply:
I need to remove that line, sorry. Yes, we need to rework that ARM_MPU so that I can disable all MPU-related stuff in a common way.

Reply:
I'll file an issue, but for now, seems we need to live with arch-specific options